src/Parallel-Tests.ps1

```diff
@@ -0,0 +1,32 @@
+param(
+    [string[]] $assemblies,
+    [string] $testFilter,
+    [string] $outDir)
+
+$xunitRunner = $(Join-Path $PSScriptRoot 'packages\xunit.runner.console.2.1.0\tools\xunit.console.exe ')
```

Comment:
It's a little long-winded, but if we want version tolerance & platform independence (join-path), then perhaps something like this?
```ps
$xunitRunner = (Get-Item $([System.IO.Path]::Combine($PSScriptRoot, "packages","xunit.runner.console.*","tools", "xunit.console.exe")) | Sort-Object FullName | Select-Object -Last 1).FullName
```

Maybe that's overkill & maybe there's a better way.